quint/src/runtime/impl/base.ts

```diff
@@ -0,0 +1,109 @@
+import { Maybe, just, none } from '@sweet-monads/maybe'
```

Comment:
These were all extracted from compilerImpl and mostly unchanged. Only new definition is `toMaybe`, which might go away if we get to convert all remaining maybe types into either types.